packages/admin-panel/src/VizBuilderApp/components/DataLibrary/component/TransformSelectedOptionWithJsonEditor.js

```diff
@@ -15,6 +15,8 @@ const getDefaultValueByType = type => {
       return [];
     case 'object':
       return {};
+    case 'boolean':
+      return false;
```

Comment:
Could have hackily fixed this one by setting this to `true`. But preferred the change that used the value from the backend